packages/libs/react-ui/src/components/Select/Select.css.ts

```diff
@@ -1,53 +1,87 @@
+import { vars } from '../../styles';
+
 import { sprinkles } from '@theme/sprinkles.css';
-import { style } from '@vanilla-extract/css';
+import { style, styleVariants } from '@vanilla-extract/css';
 
 export const containerClass = style([
   sprinkles({
-    backgroundColor: '$neutral1',
-    color: '$neutral5',
-    padding: '$2',
+    alignItems: 'stretch',
+    display: 'flex',
+    overflow: 'hidden',
+    lineHeight: '$lg',
     borderRadius: '$sm',
+    flexDirection: 'column',
+    backgroundColor: {
+      lightMode: '$white',
+      darkMode: '$background',
+    },
+    color: '$foreground',
+    borderColor: {
+      lightMode: '$white',
+      darkMode: '$gray60',
+    },
   }),
   {
     position: 'relative',
-    border: 'solid 1px',
   },
 ]);
+
 export const containerClassDisabled = style([
-  sprinkles({ pointerEvents: 'none' }),
-  { opacity: 0.5 },
+  sprinkles({
+    backgroundColor: {
+      lightMode: '$gray20',
+    },
+    color: {
+      lightMode: '$foreground',
+    },
+  }),
 ]);
 
+export const selectVariants = styleVariants({
+  default: [
+    {
+      borderBottom: `1px solid ${vars.colors.$gray30}`,
+    },
+  ],
+  flat: [
+    {
+      border: `1px solid ${vars.colors.$gray30}`,
+      background: 'transparent !important',
```

Comment:
Had to add important to overwrite the dark theme default background styling. Please let me know if there's another better way.

Reply:
You should be able to use sprinkles for this. I think it might work without the !important that way

Reply:
I tried just now but it continues to overwrite it without adding important.

Reply:
Maybe it's because you are applying the backgroundColor in the containerClass which has more specificity. I would also suggest moving any variant specific styling to the variants styling section only only keeping shared styling in the containerClass. I feel like this may fix the issue

Reply:
Ah yes, ok so I changed it to not have the background set on the containerClass and that works.